core/consensus/CMakeLists.txt

```diff
@@ -1,15 +1,19 @@
+
+
+# Use for remove "error: cast from 'const iroha::Signature*' to 'flatbuffers::uoffset_t {aka unsigned int}' loses precision"
+set(CMAKE_CXX_FLAGS "-g -std=c++1y -Wall -fPIC -fpermissive")
```

Comment:
This CMakeLists only require ```permissive``` flag. I think it flag is danger, so I shouldn't change Iroha default flag.